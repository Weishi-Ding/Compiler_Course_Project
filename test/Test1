int a : = 20;

int main(){
    int a;
    print(a);
    a = 10;
    print(a);
    
    double d;
    print(d);
    d = 2.1;
    print(a);

    string s : = "";
    print(s);
    s = "hello";
    print(s);

    bool b;
    print(b);
    b = true;
    print(b);
    
    /*literal*/
    print("hello");
    print(3.5);
    print(20);
    print(false);
}

