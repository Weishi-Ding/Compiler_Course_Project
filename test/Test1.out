0
10
0.000000
10
""
"hello"
0
1
"hello"
3.500000
20
0
